### DISCUSSION TITLE
Rental/wrap networks

### DISCUSSION THREAD

**Discussion by NathanRose84** [CMSgov/price-transparency-guide#490] [Q&A] [answered]
For rental/wrap networks , do payers have to post the rental/wrap network provider contracts in the same file as our payer networks?  It would be much easier if they could be posted separately.  @shaselton-usds

**Comment by pkincaidpopulytics** [comment]
To add on to the question -- how do we handle order of network used or is this a non-issue based on how the data is submitted?

**Comment by Devyandu** [comment, marked as answer]
With the introduction of the _index file, the payer has the flexibility to use either option depending on how their data is produced or hosted and business rules. 
It can be in one file or in multiple files. All files can be added to the in-network-files array.

The health plan I work with has decided to use multiple files. One for their national "leased" network  and one for the local network which they produce. 
The "national network" file will be produced and hosted by the plan which owns and manages the national network.

In reading other threads, I see that some payers also have network priorities which would make hosting separate files a bit tricky since there is no way to specify priority amongst multiple files. In these cases, perhaps a single file will need to produced after applying rules of network priority (if possible)

PS : I am not with CMS.

**Reply by shaselton-usds** [reply]
Thanks for answering @Devyandu! You're correct in that multiple provider networks can be provided separately that can be referenced in the in-network file.

@NathanRose84 You can find an example of referencing an external network from within the [in-network file here](https://github.com/CMSgov/price-transparency-guide/blob/59a3adc4f9d1ef6f86a306ea8714112db1805185/examples/in-network-rates/in-network-rates-fee-for-service-single-plan-sample.json#L25-L28).

**Reply by DHSDesigner** [reply]
Thanks for this feed, how is network priority established?

**Reply by skochin** [reply]
It seems there's an unaddressed issue here @shaselton-usds  - that if a payer references multiple in-network files in the TOC for a plan, and the in-network files contain overlapping providers (i.e. a provider has a contract with the "home" network and also the "wrap" network), how does the plan show which rates take precedence?  Or are they obligated to sort through all the data and choose the applicable rate in every case so as to produce a new, merged in-network-rates MRF?  This seems like it could be a processing nightmare to parse out, group together every occurrence of a provider/service/location, then compare sources to find the highest-priority rate.

**Reply by DHSDesigner** [reply]
Yes, @shaselton-usds, my question is whether we're required to show priority and also if we are required to name the networks or if we can use an internal pseudonym for those networks.

**Reply by DHSDesigner** [reply]
It's actually more complicated than that, some health plans also have leased networks (multiple)in between the home network and the wrap network. Sometimes the leased networks are one provider system, sometimes they aren't.

**Reply by DHSDesigner** [reply]
But they all have separate prices and priorities based on what plan you're using.
